WiserTaskScheduler/WiserTaskScheduler/Core/Models/WtsSettings.cs

```diff
@@ -31,7 +32,12 @@ public string Name
         /// Gets or sets the settings of the <see cref="CleanupService"/>.
         /// </summary>
         public CleanupServiceSettings CleanupService { get; set; } = new CleanupServiceSettings();
-
+        
+        /// <summary>
+        /// Gets or sets the settings of the <see cref="CleanupService"/>.
```

Comment:
The CleanupService is named while this is for the ParentsUpdateService

Reply:
done